### PR TITLE
Fix selection by user click

### PR DESCRIPTION
**Pull request by GreatWizard**
What do you think ?
I'm not enjoying to remove set on _selection... but I didn't see an other solution.

### CODE REVIEW

File: tests/integration/components/simple-select-test.js [hunk 1]
```diff
@@ -66,6 +66,22 @@ test('It changes the selected option when the "selection" attribute is updated',
...
+  this.set('content', [
+    { id: 1, title: 'One'},
+    { id: 2, title: 'Two'},
+    { id: 3, title: 'Three'}
+  ]);
+
+  this.set('selection', this.get('content.2'));
+  this.render(hbs`{{simple-select content=content selection=selection}}`);
+  assert.equal(this.$('select').val(), 3);
+  this.$('select option:nth-child(2)').attr('selected', 'selected').trigger('change');
```
Comment (xcambar) [inline tests/integration/components/simple-select-test.js]: The idiomatic way to write this is:

```js
fillIn('select', 'Two');
// or
fillIn('select', this.get('content.1.title'));
```

**Comment by xcambar** [comment]
Pretty nice. Thx for hunting it down.

**Comment by xcambar** [comment]
Feel free to merge after reviewing my comment :shipit:

**Comment by GreatWizard** [comment]
I try fillIn yesterday but it doesn"t work on select.
For me it's good like this!

**Pull request merged**